tutorial_R/R_rainclouds.R

```diff
@@ -3,17 +3,8 @@
 ### It relies largely on code previously written by David Robinson 
 ### (https://gist.github.com/dgrtwo/eb7750e74997891d7c20) and ggplot2 by H Wickham
 
-
-#check if required packages are installed
-packages <- c("cowplot", "readr", "ggplot2" , "dplyr", "lavaan", "smooth", "Hmisc")
-if (length(setdiff(packages, rownames(installed.packages()))) > 0) {
-  install.packages(setdiff(packages, rownames(installed.packages())))  
-}
-
 #Load packages
```

Comment:
@ha0ye - this is a question highly driven by the fact that I don't know R very well - so sorry if it's really basic!

What's the workflow to make sure that the packages are all installed for the local user? I know the `install.R` has the files we need and that will work for binder (thank you!) but how will the local user make sure they have everything?

The goal of this if statement was to make sure that all the packages were installed locally. Maybe it isn't needed but I'd lvoe to make sure this is clear to the user 😸

Reply:
Ah yep, I'll include it again at the top of the Rmarkdown for clarity.

Reply:
Ah, yeah, I hadn't considered that. I'll add that back in as a step in the R markdown.

Reply:
So sorry to be mega difficult - I think it makes more sense to add these packages at the top of this file because that way they're there when someone uses this file for their own purposes. Do you feel strongly about not having this at the top of the .R file?